devops/prod/values.yaml

```diff
@@ -182,4 +183,4 @@ iam-cache-server-helm:
       IPFS_CLUSTER_USER: AgBUy6GBiFTMwz4Q1jjlEzy7lt4yfyMpYjhs3SeGA+X0Qo34ERmy+uxTlkraCK/uq7M5mcLsH609wa4ubEx2aV5jQCaVZ5rhnEW71KCRfkGUIcip1WbdVTvrtESADPUadFf7pEXm+OvJMQPNPMjQ4uyS57M/Zl16/oNltlsffLRanAgZohLMPXcLi4XC4ZUTSI1erHWMuL2pR9s6+edHEtd+7PEa7jbFdfZdeDPdnh0Xgf+3XXM6wXqykaaCSG/lCWIpiqrklSrKf+jt8HC2vJ/R+iK2RefXjH1LlR8JiwEzUaa6vbLBx0UVGAe9ncShWLXZtzKl+aqChqpiYHCCuD4h5bEKZK4JTpPsojQuoi8TutGTQPlislakzN+20DcAqdZ7soNmOVvhKyNwcLB5ycbK7wW5XB0k2MjsOxHBjLF28dFiroCTjskJgUxot1EPo8wSdssBlGxPYu9HsVG7jH0iw/RCDoepJdx47t74TPOpdaIVvBYYu6TJy4ohN1mED7c4kvXGEk1YOcZXCqXf/Grn3/rLUoAFUOrLRUFdOgXcz5FCbF50azES/Oo59LCCxHCNWx+xcJl3+QmCme+X7h6RdLDUU5TFlRoZrAoVtBQGrkQKVWZzuRYmjHm3ZVXJJfzPqzc+7pvQ1l1dIokG6NHzUJ2iIRv3qkagyH7BlQnoj0tOsSnWuReW1QHIboOWAL96qyfmadIxZJu94hEv/bIM
-      IPFS_CLUSTER_PASSWORD: AgCncDxtjXgBmdaMT+a7NN8PIZQaNH7PJ/C5RmGkcXwYUsS9JONjWqBny+7OEPM9Y82DaYGrsH3LwBrCG2wJ8ja5zelHfrkl4ue9HnplpbhnG9eigMRer0pFHbwdrqagGZNI5EOZXEQNZb3uy/BUFbvB+dWvEbYnNz4UYHU/3hQAAbuQ81VniTRmmSb6kbNN58mHVHCwbC1kjQsssQ1cVE5aTSLZFfxgb1zIvgrhZKKWmG0ygZDV78XMHZ8fCvBACcIlt9Suy9JdpJMZG8omvtYwF6wvGj7w1DiZwr67Pmt0i2959MgFA6nIYxo8mJbEaDaAnTYAheiNHWnTXxVXCQDECN5xNkd81tNfj0wMQa8CqgH0KbH/lJCFSyruKA4Up/7h7xhU/SG9nx8DyzV1g0phxkggHcXBeFGeupwHS6d5O80N9ThxVMLznrHgpyqdz4ZZORtGjvPoH+3ehLg22CbZasPhEPzkfT4wg0NBRNqf7YUqEz+WS41CQVHL/PqON39YzEkqy0MZhAdhSWKjSopdQBfZxji5DujbofX17qSMD3IWsTpF3IV3pRCZAqAmH77R93FxJWBtrJiUztGf4Po8fiYScusNiEORhFru0x1+avH8aHxbNvDfsPGN5NOAsoQ8AD6SAvPDwTacl8YydccAv71To+sNQYP+qgmKk4ir6Nn5sB/tAXDLzvfx9QHU+j6jgY48abU4q2ZKA2GKG6po/lu+jawugA==
\ No newline at end of file
+      IPFS_CLUSTER_PASSWORD: AgCncDxtjXgBmdaMT+a7NN8PIZQaNH7PJ/C5RmGkcXwYUsS9JONjWqBny+7OEPM9Y82DaYGrsH3LwBrCG2wJ8ja5zelHfrkl4ue9HnplpbhnG9eigMRer0pFHbwdrqagGZNI5EOZXEQNZb3uy/BUFbvB+dWvEbYnNz4UYHU/3hQAAbuQ81VniTRmmSb6kbNN58mHVHCwbC1kjQsssQ1cVE5aTSLZFfxgb1zIvgrhZKKWmG0ygZDV78XMHZ8fCvBACcIlt9Suy9JdpJMZG8omvtYwF6wvGj7w1DiZwr67Pmt0i2959MgFA6nIYxo8mJbEaDaAnTYAheiNHWnTXxVXCQDECN5xNkd81tNfj0wMQa8CqgH0KbH/lJCFSyruKA4Up/7h7xhU/SG9nx8DyzV1g0phxkggHcXBeFGeupwHS6d5O80N9ThxVMLznrHgpyqdz4ZZORtGjvPoH+3ehLg22CbZasPhEPzkfT4wg0NBRNqf7YUqEz+WS41CQVHL/PqON39YzEkqy0MZhAdhSWKjSopdQBfZxji5DujbofX17qSMD3IWsTpF3IV3pRCZAqAmH77R93FxJWBtrJiUztGf4Po8fiYScusNiEORhFru0x1+avH8aHxbNvDfsPGN5NOAsoQ8AD6SAvPDwTacl8YydccAv71To+sNQYP+qgmKk4ir6Nn5sB/tAXDLzvfx9QHU+j6jgY48abU4q2ZKA2GKG6po/lu+jawugA==
```

Comment:
Another minor point, but probably this line shouldn't be changed. It seems there is no actual change, but maybe better not to take a risk and have an extra entry in the git history for this line.

Reply:
Probably line ending here was changed. I tried to reset this file to develop but it didn't help. I added .gitiattributes to prevent such issues

Reply:
Ok, thanks